src/Misc/layoutbin/installdependencies.sh

```diff
@@ -232,13 +240,46 @@ then
                 exit 1
             fi
         else
-            echo "Can't detect current OS type base on /etc/os-release."
+            echo "Can't detect current OS type based on /etc/os-release."
             print_errormessage
             exit 1
         fi
     fi
+elif [ -e /etc/redhat-release ]
+# RHEL6 doesn't have an os-release file defined, read redhat-release instead
+then
+    redhatRelease=$(</etc/redhat-release)
+    if [[ $redhatRelease == "CentOS release 6."* || $redhatRelease == "Red Hat Enterprise Linux Server release 6."* ]]
+    then
+        echo "The current OS is Red Hat Enterprise Linux 6 or Centos 6"
+
+        # Install known dependencies, as a best effort.
+        # The remaining dependencies are covered by the GitHub doc that will be shown by `print_rhel6message`
+        command -v yum
+        if [ $? -eq 0 ]
+        then
+            yum install -y openssl krb5-libs zlib
+            if [ $? -ne 0 ]
+            then                    
+                echo "'yum' failed with exit code '$?'"
+                print_rhel6message
```

Comment:
we might need 2 redhat 6 message.
on succeed, print out you need to install extra dependencies yourself.
on failure, pint out dependencies install failed, do yourself. 

:D

Reply:
Sounds good, I'll implement it. The error message for RHEL6 would look like this:

```sh
function print_rhel6errormessage() 
{
    echo "We couldn't install dotnet core dependencies"
    echo "You can manually install all required dependencies based on following documentation"
    echo "https://docs.microsoft.com/en-us/dotnet/core/linux-prerequisites?tabs=netcore2x"
    echo "In addition, there are some dependencies which require manual installation. Please follow this documentation" 
    echo "https://github.com/dotnet/core/blob/master/Documentation/build-and-install-rhel6-prerequisites.md"
}
```